packages/pricing/src/models/price.ts

```diff
@@ -129,10 +129,14 @@ export default class Price {
   @BeforeCreate()
   onCreate() {
     this.id = generateEntityId(this.id, "price")
+    this.price_set_id ??= this.price_set?.id!
```

Comment:
thought(non-blocking): When we add these assignments to the hooks, I get a slight feeling that we should be doing something different. With our approach, this becomes a very important part of the model definition, yet easily forgotten. And it is not the easiest to debug in case you accidentally left them out. Maybe there is a better way.

Have you had the same thought, or is this just me? Also, this is not something we need to act on, but I was just curious.

Reply:
For now i think it is the easiest approach. In terms of debug, I didn't find it particularly difficult on my side, usually you have something telling you the foreign key is missing. Do you have something in mind?